File: BaekJoon Online Judge/C++/Step08-Math_1 Mathematical Thinking/1712.cpp
//
//  1712.cpp
//  Step08-Math_1 Mathematical Thinking
//
//  Created by Yebin Kim on 2019/10/23.
//  Copyright © 2019 김예빈. All rights reserved.
//

#include <iostream>

using namespace std;

// Time Complexity O(1)
int main() {
    cin.tie(NULL);
    ios::sync_with_stdio(false);
    
    int a, b, c;
    int answer;
    
    cin >> a >> b >> c;
    
    if(c - b > 0)
        answer = a / (c - b) + 1;
    else
        answer = -1;
        
    cout << answer;
    
    return 0;
}

File: BaekJoon Online Judge/Swift/etc/1712.swift
//
//  1712.swift
//  Step08-Math_1 Mathematical Thinking
//
//  Created by Yebin Kim on 2019/11/13.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

let num = readLine()!.split(separator: " ").map{ Int($0)! }
let A = num[0]
let B = num[1]
let C = num[2]

if C - B <= 0 {
    print(-1)
} else {
    print(A / (C - B) + 1)
}
